openedx/features/partners/helpers.py

```diff
@@ -107,3 +109,13 @@ def get_partner_from_user(user):
 
 def user_has_performance_access(user, partner):
     return bool(partner and user.has_perm('partners.%s' % PERFORMANCE_PERM_FRMT.format(slug=partner.slug)))
+
+
+def get_partner_registration_countries(partner_slug):
+    sorted_countries = sorted(COUNTRIES.items(), key=operator.itemgetter(1))
+    top_registration_countries = PARTNERS_TOP_REGISTRATION_COUNTRIES.get(partner_slug, None)
+    if top_registration_countries:
+        top_countries = [(country[0], COUNTRIES.get(country[0], None)) for country in top_registration_countries]
```

Comment:
If we are extracting countries from `COUNTRIES` can we update constants like the following code? This looks logical because `COUNTRIES` already have country names.

`PARTNERS_TOP_REGISTRATION_COUNTRIES = ('NG', 'IN', 'PK', 'KE', 'GH')`

If we go with this suggestion, we can make it a utility function.